src/index.html

```diff
@@ -0,0 +1,1302 @@
+      </div>
+      <div class="nav-bars">
+        <div class="left-nav">
+          <ul class="flex">
+            <li>
+              <a href="#">
+                News
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Events
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Business
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                AMAs
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Spaces
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Terms & Conditions
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="right-nav">
+          <ul class="flex">
+            <li>
+              <a href="#">
+                About
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Advertise
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Jobs
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                Contact
+              </a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="flex bottom-nav">
+          <ul class="flex">
+            <li>
+              <a href="#">
+                LATEST
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                HARD FORK
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                PLUGGED
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                README
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                GROWTH QUATERS
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                NEURAL
+              </a>
+            </li>
+          </ul>
+          <div class="flex search-aside-buttons">
+            <a href="#">
+              <i class="fas fa-search"></i>
+            </a>
+            <button class="nav-btn" type="button">
+              <i class="fas fa-bars"></i>
+            </button>
+          </div>
+        </div>
+
+      </div>
+
+    </nav>
+    <!-- START of header section -->
+
+    <header>
+
+      <div class="header-articles">
+        <div class="left-box">
+          <img src="../src/assets/svg/TNW-white.svg" alt="main-logo">
+          <p class="date">
+            Wednesday - July 29, 2020
+          </p>
+          <a class="head-box-text" href="#">
+            <h6>
+              INSIGHTS
+            </h6>
+            <h3 class="main-head-text">
+              Satoshi Nakaboto: 'Grayscale exceeds $5B in Bitcoin under managment'
+            </h3>
+            <span>
```

Comment:
#### Mobile screen size

- [ ] The background images are not really responsive. They break at 400px. Since it is a clone, your project should behave the same way. Also, pay attention to the font-sizes and font-colors.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89058369-95500c00-d335-11ea-94ef-b39d334e916b.png) |![image](https://user-images.githubusercontent.com/61479648/89058419-a436be80-d335-11ea-9cd7-d6e9dec7dbde.png)|

#### Medium Screen size

- [ ] The TNW logo and the date should disappear (don't mind the navbar in the screenshot)

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89058603-f37cef00-d335-11ea-9654-d125820ef8f8.png)| ![image](https://user-images.githubusercontent.com/61479648/89058626-0099de00-d336-11ea-8948-888ad4f3a973.png) |